overlord/ifacestate/handlers.go

```diff
@@ -1068,13 +1110,13 @@ func (m *InterfaceManager) doAutoConnect(task *state.Task, _ *tomb.Tomb) error {
 		}
 	}
 
-	// Create connect tasks and interface hooks
-	for _, conn := range newconns {
-		ts, err := connect(st, conn.PlugRef.Snap, conn.PlugRef.Name, conn.SlotRef.Snap, conn.SlotRef.Name, connectOpts{AutoConnect: true})
-		if err != nil {
-			return fmt.Errorf("internal error: auto-connect of %q failed: %s", conn, err)
-		}
-		autots.AddAll(ts)
+	setupProfiles := st.NewTask("setup-profiles", fmt.Sprintf(i18n.G("Setup snap %q (%s) security profiles"), snapsup.InstanceName(), snapsup.Revision()))
+	setupProfiles.Set("snap-setup", snapsup)
```

Comment:
not sure why is this task is created here an not the in the helper?  also the summary should relate to/mention auto-connect

Reply:
Fair point, no deep reason really. Changed. Thank you.